xray/commands/audit/gradle.go

```diff
@@ -12,6 +12,7 @@ import (
 
 type AuditGradleCommand struct {
 	serverDetails          *config.ServerDetails
+	printFormat            PrintFormat
```

Comment:
printFormat            PrintFormat
-->
outputFormat            OutputFormat

Let's make this naming change in all commands.